sdcm/nemesis.py

```diff
@@ -1762,13 +1762,13 @@ def streaming_task_thread(nodetool_task='rebuild'):
 
         def is_streaming_started():
             stream_pattern = "range_streamer - Unbootstrap starts|range_streamer - Rebuild starts"
-            streaming_logs = self.target_node.search_database_log(stream_pattern, start_from_beginning=False)
+            streaming_logs = self.target_node.search_database_log(stream_pattern, start_from_beginning=False, publish_events=False)
```

Comment:
can you also change the default in `search_database_log()`  to be ERROR or WARNING. 
since now CRITICAL halts the tests...

Reply:
I just updated the PR to make the event severity configurable.
The default value for search_database_log() is ERROR.

Reply:
that's good, lets get this merge. please address the pre-commit issues